### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by TemesgenA**
### Feature added
- Added popup windows that show project details
#### Lessons learned
- basic JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- JavaScript events.
- Understand how to parse a Figma design to create a UI

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -10,3 +10,186 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
+    </a>
+    <a href="https://temesgena.github.io/Portfolio-mobile/">
+      <button class="btn modal-btn" type="submit">
+        See Source
+        <img src="./Assets/vectors/source-code-icon.svg" class="modal-vectors" alt="source-code"/>
+      </button>
+    </a>
+  </div>
+</div>
+`;
```
Comment (williamrolando88) [inline script.js]: - [ ] As the project requirement says, you need to open the popup with the details of the selected project, right now when I click on the "See project" button, it only renders a static popup that has the same content for all projects.

File: script.js [hunk 2]
```diff
@@ -10,3 +10,186 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
...
+    projectDiv.classList.add('card-reverse');
+  }
+});
+
+const modalContainer = document.querySelector('.modal-container');
+const modal = document.createElement('div');
+modalContainer.append(modal);
+modal.classList.add('modal');
+
+modal.innerHTML = `
```
Comment (williamrolando88) [inline script.js]: - [ ] [Optional] To improve users experience, please make sure that when scrolling down the popup window, the close button don't disappear from screen:
image

**Review by williamrolando88 — CHANGES_REQUESTED** [review]
Hi @TemesgenA , 

### Please make sure to not delete or create new PR for oncoming reviews, the system says that this is not the first review for this exercise.


 ## Highlights

- Nice job implementing the responsive design

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TemesgenA** [comment]
Hello there,
This is my final code review submission. And no extra reviews left for me.
So please try to consider that.
Kindly.

File: style.css [hunk 3]
```diff
@@ -885,10 +885,20 @@ hr {
 }
 
 .modal-close {
```
Comment (Ismailco) [inline style.css:887]: - [ ] [OPTIONAL] You should implement the above requirements in all repositories in your pair-programming group, kindly in the future pay attention to this requirement

**Review by Ismailco — APPROVED** [review]
### Hi @TemesgenA, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (TemesgenA) [reply style.css:887]: @ismailco I will. Thanks, man.

**Pull request merged**